cmd/http/main.go

```diff
@@ -28,20 +28,20 @@ import (
 	brokerstracing "github.com/absmach/magistrala/pkg/messaging/brokers/tracing"
 	"github.com/absmach/magistrala/pkg/messaging/handler"
 	"github.com/absmach/magistrala/pkg/uuid"
-	mproxy "github.com/absmach/mproxy/pkg/http"
+	"github.com/absmach/mproxy"
+	mproxyhttp "github.com/absmach/mproxy/pkg/http"
 	"github.com/absmach/mproxy/pkg/session"
-	"github.com/caarlos0/env/v10"
+	"github.com/caarlos0/env/v11"
 	"go.opentelemetry.io/otel/trace"
 	"golang.org/x/sync/errgroup"
 )
 
 const (
 	svcName        = "http_adapter"
-	envPrefix      = "MG_HTTP_ADAPTER_"
+	envPrefixHTTP  = "MG_HTTP_ADAPTER_"
```

Comment:
Revert to using `envPrefix`
```suggestion
	envPrefix  = "MG_HTTP_ADAPTER_"
```